calibr8/tests.py

```diff
@@ -1285,6 +1285,58 @@ def test_fit_pygmo(self, caplog):
         numpy.testing.assert_array_equal(em.cal_dependent, y)
         pass
 
+    def test_fit_scipy_global(self, caplog):
+        numpy.random.seed(2345)
```

Comment:
Where are you using this seed? It's better not to set it gloablly (you could forget to reset it, but rather instantiate a random number generator with the seed and use this for all further options):

`rng = rng = numpy.random.RandomState(seed)` and later sth like `rng.permutation` or `rng.uniform`